nexus/src/app/bgp.rs

```diff
@@ -107,13 +108,29 @@ impl super::Nexus {
             };
 
             for r in &router_info {
-                for (addr, info) in &r.peers {
-                    let Ok(addr) = addr.parse() else {
+                let asn = r.asn;
+
+                let peers = match client.get_neighbors(asn).await {
+                    Ok(result) => result.into_inner(),
+                    Err(e) => {
+                        error!(
+                        self.log,
+                        "failed to get peers for asn {asn} from {switch}: {e}"
+                    );
+                        continue;
```

Comment:
is this `continue` really right? feels like this should error out -- what semantics do we desire?